commisery/cli/__init__.py

```diff
@@ -175,10 +175,12 @@ def _read_commit_message():
 
         answers = inquirer.prompt(questions)
 
-        answers["scope"] = answers.get("scope") if answers.get("scope") else None
+        answers["scope"] = answers.get("scope") if answers.get("scope", None) else None
         answers["type"] = answers.get("type").split(":")[0]
-        answers["breaking_change"] = "!" if answers.get("breaking_change") else ""
-        answers["body"] = (os.linesep + answers.get("body")).split(os.linesep)
+        answers["breaking_change"] = "!" if answers.get("breaking_change", "No") == "Yes" else ""
+
+        if answers.get("body"):
+            answers["body"] = (os.linesep + answers.get("body")).split(os.linesep)
```

Comment:
You only need `os.linesep` when you're performing I/O on *binary* files. Within Python (or within anything built on top of the C library for that matter) line terminators are converted to "\n" by the file read functions (and back to `os.linesep` by the write functions) for files opened in text mode. So just stick to `"\n"`. Also, lines are special: `str.splitlines(self, /, keepends=False)`

```suggestion
            answers["body"] = ("\n" + answers.get("body")).splitlines()
```